bot/resources/evergreen/py_topics.yaml

```diff
@@ -45,19 +54,26 @@
     - What feature would you be the most interested in making?
     - What feature would you like to see added to the library? what feature in the library do you think is redundant?
     - Do you think there's a way in which Discord could handle bots better?
+    - What's one feature you wish more developers had in their bots?
+
+# editors-ides
+813178633006350366:
+    - What's your current text editor/IDE, and what functionality do you like about it the most when programming in Python?
```

Comment:
This question is quite wordy, but since it's just moving and old question, I won't complain if you leave it as is. If you're  looking for ways to shorten it, I think we can start with lobbing of `when programming in Python`.